File: scratch/check_captions.py
import os
import re

files_to_check = [
    r'd:\Dropbox\03_super planning\00_슈퍼플래닝\2026_uperplanning_website\implementation\ux_design\index.html',
    r'd:\Dropbox\03_super planning\00_슈퍼플래닝\2026_uperplanning_website\implementation\ux_design_cleanroom\index.html',
    r'd:\Dropbox\03_super planning\00_슈퍼플래닝\2026_uperplanning_website\implementation\ux_design.html',
    r'd:\Dropbox\03_super planning\00_슈퍼플래닝\2026_uperplanning_website\implementation\ux_design_cleanroom.html',
    r'd:\Dropbox\03_super planning\00_슈퍼플래닝\2026_uperplanning_website\implementation\ux-design.html',
    r'd:\Dropbox\03_super planning\00_슈퍼플래닝\2026_uperplanning_website\implementation\ux_plan\index.html',
]

for filepath in files_to_check:
    if os.path.exists(filepath):
        with open(filepath, 'r', encoding='utf-8', errors='ignore') as f:
            content = f.read()
        print(f"=== File: {filepath} ===")
        # Find 3. UX컨설팅 or consulting section
        idx = content.find('3. UX컨설팅')
        if idx != -1:
            snippet = content[idx:idx+2500]
            snippet_clean = re.sub(r'src="data:image/[^"]+"', 'src="BASE64"', snippet)
            print(snippet_clean)
        else:
            print("Section '3. UX컨설팅' not found")

File: scratch/check_captions.js
const fs = require('fs');

const files_to_check = [
    'd:\\Dropbox\\03_super planning\\00_슈퍼플래닝\\2026_uperplanning_website\\implementation\\ux_design\\index.html',
    'd:\\Dropbox\\03_super planning\\00_슈퍼플래닝\\2026_uperplanning_website\\implementation\\ux_design_cleanroom\\index.html',
    'd:\\Dropbox\\03_super planning\\00_슈퍼플래닝\\2026_uperplanning_website\\implementation\\ux_design.html',
    'd:\\Dropbox\\03_super planning\\00_슈퍼플래닝\\2026_uperplanning_website\\implementation\\ux_design_cleanroom.html',
    'd:\\Dropbox\\03_super planning\\00_슈퍼플래닝\\2026_uperplanning_website\\implementation\\ux-design.html',
    'd:\\Dropbox\\03_super planning\\00_슈퍼플래닝\\2026_uperplanning_website\\implementation\\ux_plan\\index.html',
];

files_to_check.forEach(filepath => {
    if (fs.existsSync(filepath)) {
        const content = fs.readFileSync(filepath, 'utf8');
        let idx = content.indexOf('<h2 id="consulting">3. UX컨설팅</h2>');
        if (idx === -1) idx = content.indexOf('3. UX컨설팅</h2>');
        if (idx !== -1) {
            let nextH2 = content.indexOf('<h2', idx + 10);
            if (nextH2 === -1) nextH2 = idx + 3000;
            let snippet = content.substring(idx, nextH2);
            let snippet_clean = snippet.replace(/src="data:image\/[^"]+"/g, 'src="BASE64"');
            console.log(`=== File: ${filepath} ===`);
            console.log(snippet_clean);
        }
    }
});
